src/about_dialog.js

```diff
@@ -0,0 +1,33 @@
+const { BrowserWindow } = require('electron');
+const path = require('path');
+
+const width = process.platform === 'darwin' ? 510 : 530;
+const height = process.platform === 'darwin' ? 325 : 330;
```

Comment:
I think making this 330 for all platforms will probably look better - windows and linux both have a scroll bar on the About dialog currently.

Reply:
I kept it the same that it was figuring the style changes come later.